src/components/Dashboard.vue

```diff
@@ -66,6 +91,16 @@ Vue.component('requirements', Requirements);
 Vue.component('bottombar', BottomBar);
 Vue.component('navbar', NavBar);
 Vue.component('onboarding', Onboarding);
+Vue.component('tourwindow', TourWindow);
+
+const tour = introJs();
+tour.setOption('exitOnEsc', 'false');
+tour.setOption('doneLabel', 'Finish');
+tour.setOption('skipLabel', 'Skip This Tutorial');
+tour.setOption('nextLabel', 'Next');
+tour.setOption('exitOnOverlayClick', 'false');
+
+const surfing = require('../assets/images/surfing.svg');
```

Comment:
How come the season emojis use require?

Reply:
That's probably just what was intuitive to whoever wrote it and we didn't catch it before. I personally think leaving it as require would be better for now since it is consistent with the other emojis like you mentioned - but I can make a Monday item to replace all requires with imports later.

I just tested it by the way, and a line like this works fine. 
`import surfing from '@/assets/images/surfing.svg';`